src/os_mswin.c

```diff
@@ -2992,6 +2993,9 @@ get_logfont(
 	    case L'w':
 		lf->lfWidth = points_to_pixels(p, &p, FALSE, (long_i)printer_dc);
 		break;
+	    case L'W':
+		lf->lfWeight = wcstol(p, &p, 0);
```

Comment:
Isn't it better to use `10` for the 3rd argument?
We don't need to support octal or hex numbers here.